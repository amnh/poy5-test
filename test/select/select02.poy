(* 'select' terminals using terminals file without identifiers *)
set(seed:1)
read ("1.fas", "30.ss", aminoacids:("40.aa"))
select("term12.txt")
build(2)
report(data)