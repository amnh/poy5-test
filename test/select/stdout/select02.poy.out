                  
Included Excluded
t5       t9
t49      t8
t45      t7
t40      t6
t35      t48
t30      t47
t25      t46
t20      t44
t2       t43
t15      t42
t10      t41
t1       t4
         t39
         t38
         t37
         t36
         t34
         t33
         t32
         t31
         t3
         t29
         t28
         t27
         t26
         t24
         t23
         t22
         t21
         t19
         t18
         t17
         t16
         t14
         t13
         t12
         t11
Data
    


Taxa
    
    Total 12
                                    
 Name         Code          
t1           1            {}
t10          10           {}
t15          15           {}
t2           2            {}
t20          20           {}
t25          25           {}
t30          30           {}
t35          35           {}
t40          40           {}
t45          45           {}
t49          49           {}
t5           5            {}

Synonyms
    
    Total 0



Ignored taxa
    t9;
t8;
t7;
t6;
t48;
t47;
t46;
t44;
t43;
t42;
t41;
t4;
t39;
t38;
t37;
t36;
t34;
t33;
t32;
t31;
t3;
t29;
t28;
t27;
t26;
t24;
t23;
t22;
t21;
t19;
t18;
t17;
t16;
t14;
t13;
t12;
t11;

Characters
    Non Additive
    
    Total 0



Additive
    
    Total 144
                                                                                                                                                                  
 File Source         Name         Weight         Missing Symbol         Match Symbol         Gap Symbol         Ignore         Case Sentive         Range
30.ss               30.ss:0      1.             ?                                           -                  false          true                 {1, 2, 3, 4, }
30.ss               30.ss:1      1.             ?                                           -                  false          true                 {1, }
30.ss               30.ss:2      1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:3      1.             ?                                           -                  false          true                 {2, }
30.ss               30.ss:4      1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:5      1.             ?                                           -                  false          true                 {1, 2, 3, 4, }
30.ss               30.ss:6      1.             ?                                           -                  false          true                 {1, }
30.ss               30.ss:7      1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:8      1.             ?                                           -                  false          true                 {1, }
30.ss               30.ss:9      1.             ?                                           -                  false          true                 {1, 3, }
30.ss               30.ss:10     1.             ?                                           -                  false          true                 {2, }
30.ss               30.ss:11     1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:12     1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:13     1.             ?                                           -                  false          true                 {1, }
30.ss               30.ss:14     1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:15     1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:16     1.             ?                                           -                  false          true                 {2, }
30.ss               30.ss:17     1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:18     1.             ?                                           -                  false          true                 {4, }
30.ss               30.ss:19     1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:20     1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:21     1.             ?                                           -                  false          true                 {2, }
30.ss               30.ss:22     1.             ?                                           -                  false          true                 {1, 2, 3, 4, }
30.ss               30.ss:23     1.             ?                                           -                  false          true                 {2, }
30.ss               30.ss:24     1.             ?                                           -                  false          true                 {1, 3, }
30.ss               30.ss:25     1.             ?                                           -                  false          true                 {2, }
30.ss               30.ss:26     1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:27     1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:28     1.             ?                                           -                  false          true                 {1, 2, 3, 4, }
30.ss               30.ss:29     1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:30     1.             ?                                           -                  false          true                 {1, 3, }
30.ss               30.ss:31     1.             ?                                           -                  false          true                 {2, }
30.ss               30.ss:32     1.             ?                                           -                  false          true                 {2, }
30.ss               30.ss:33     1.             ?                                           -                  false          true                 {1, 3, }
30.ss               30.ss:34     1.             ?                                           -                  false          true                 {1, }
30.ss               30.ss:35     1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:36     1.             ?                                           -                  false          true                 {1, 4, }
30.ss               30.ss:37     1.             ?                                           -                  false          true                 {2, }
30.ss               30.ss:38     1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:39     1.             ?                                           -                  false          true                 {1, 2, 3, 4, }
30.ss               30.ss:40     1.             ?                                           -                  false          true                 {2, }
30.ss               30.ss:41     1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:42     1.             ?                                           -                  false          true                 {2, }
30.ss               30.ss:43     1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:44     1.             ?                                           -                  false          true                 {1, 2, }
30.ss               30.ss:45     1.             ?                                           -                  false          true                 {1, 4, }
30.ss               30.ss:46     1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:47     1.             ?                                           -                  false          true                 {1, 3, }
30.ss               30.ss:48     1.             ?                                           -                  false          true                 {2, }
30.ss               30.ss:49     1.             ?                                           -                  false          true                 {1, 2, }
30.ss               30.ss:50     1.             ?                                           -                  false          true                 {1, 3, 4, }
30.ss               30.ss:51     1.             ?                                           -                  false          true                 {1, }
30.ss               30.ss:52     1.             ?                                           -                  false          true                 {3, }
30.ss               30.ss:53     1.             ?                                           -                  false          true                 {1, 3, }
30.ss               30.ss:54     1.             ?                                           -                  false          true                 {2, }
30.ss               30.ss:55     1.             ?                                           -                  false          true                 {1, 3, }
30.ss               30.ss:56     1.             ?                                           -                  false          true                 {1, 3, }
30.ss               30.ss:57     1.             ?                                           -                  false          true                 {1, 2, }
30.ss               30.ss:58     1.             ?                                           -                  false          true                 {3, }
30.ss               30.ss:59     1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:60     1.             ?                                           -                  false          true                 {2, }
30.ss               30.ss:61     1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:62     1.             ?                                           -                  false          true                 {1, 3, }
30.ss               30.ss:63     1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:64     1.             ?                                           -                  false          true                 {1, 4, }
30.ss               30.ss:65     1.             ?                                           -                  false          true                 {2, 3, 4, }
30.ss               30.ss:66     1.             ?                                           -                  false          true                 {2, }
30.ss               30.ss:67     1.             ?                                           -                  false          true                 {2, 3, }
30.ss               30.ss:68     1.             ?                                           -                  false          true                 {2, }
30.ss               30.ss:69     1.             ?                                           -                  false          true                 {1, 3, }
30.ss               30.ss:70     1.             ?                                           -                  false          true                 {1, 3, }
30.ss               30.ss:71     1.             ?                                           -                  false          true                 {2, 3, }
30.ss               30.ss:72     1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:73     1.             ?                                           -                  false          true                 {1, 2, }
30.ss               30.ss:74     1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:75     1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:76     1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:77     1.             ?                                           -                  false          true                 {1, 2, 3, }
30.ss               30.ss:78     1.             ?                                           -                  false          true                 {3, 4, }
30.ss               30.ss:79     1.             ?                                           -                  false          true                 {3, 4, }
30.ss               30.ss:80     1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:81     1.             ?                                           -                  false          true                 {1, 2, }
30.ss               30.ss:82     1.             ?                                           -                  false          true                 {2, 3, }
30.ss               30.ss:83     1.             ?                                           -                  false          true                 {1, 3, 4, }
30.ss               30.ss:84     1.             ?                                           -                  false          true                 {1, 2, 3, 4, }
30.ss               30.ss:85     1.             ?                                           -                  false          true                 {1, 3, }
30.ss               30.ss:86     1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:87     1.             ?                                           -                  false          true                 {1, 2, 3, 4, }
30.ss               30.ss:88     1.             ?                                           -                  false          true                 {1, 3, }
30.ss               30.ss:89     1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:90     1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:91     1.             ?                                           -                  false          true                 {1, 3, }
30.ss               30.ss:92     1.             ?                                           -                  false          true                 {1, 3, }
30.ss               30.ss:93     1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:94     1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:95     1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:96     1.             ?                                           -                  false          true                 {1, 2, 3, 4, }
30.ss               30.ss:97     1.             ?                                           -                  false          true                 {1, 2, }
30.ss               30.ss:98     1.             ?                                           -                  false          true                 {1, 2, 3, 4, }
30.ss               30.ss:99     1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:100    1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:101    1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:102    1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:103    1.             ?                                           -                  false          true                 {1, 2, }
30.ss               30.ss:104    1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:105    1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:106    1.             ?                                           -                  false          true                 {1, 2, 3, 4, }
30.ss               30.ss:107    1.             ?                                           -                  false          true                 {2, }
30.ss               30.ss:108    1.             ?                                           -                  false          true                 {1, 3, }
30.ss               30.ss:109    1.             ?                                           -                  false          true                 {1, 3, }
30.ss               30.ss:110    1.             ?                                           -                  false          true                 {2, }
30.ss               30.ss:111    1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:112    1.             ?                                           -                  false          true                 {1, 4, }
30.ss               30.ss:113    1.             ?                                           -                  false          true                 {2, 3, }
30.ss               30.ss:114    1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:115    1.             ?                                           -                  false          true                 {1, 3, }
30.ss               30.ss:116    1.             ?                                           -                  false          true                 {4, }
30.ss               30.ss:117    1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:118    1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:119    1.             ?                                           -                  false          true                 {1, 2, 3, }
30.ss               30.ss:120    1.             ?                                           -                  false          true                 {1, 2, 3, 4, }
30.ss               30.ss:121    1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:122    1.             ?                                           -                  false          true                 {1, 3, 4, }
30.ss               30.ss:123    1.             ?                                           -                  false          true                 {1, 2, }
30.ss               30.ss:124    1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:125    1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:126    1.             ?                                           -                  false          true                 {1, 2, 3, }
30.ss               30.ss:127    1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:128    1.             ?                                           -                  false          true                 {3, }
30.ss               30.ss:129    1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:130    1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:131    1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:132    1.             ?                                           -                  false          true                 {1, 3, 4, }
30.ss               30.ss:133    1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:134    1.             ?                                           -                  false          true                 {1, 3, }
30.ss               30.ss:135    1.             ?                                           -                  false          true                 {2, }
30.ss               30.ss:136    1.             ?                                           -                  false          true                 {1, 3, }
30.ss               30.ss:137    1.             ?                                           -                  false          true                 {2, 4, }
30.ss               30.ss:138    1.             ?                                           -                  false          true                 {1, 4, }
30.ss               30.ss:139    1.             ?                                           -                  false          true                 {1, 2, 3, 4, }
30.ss               30.ss:140    1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:141    1.             ?                                           -                  false          true                 {2, }
30.ss               30.ss:142    1.             ?                                           -                  false          true                 {1, 2, 4, }
30.ss               30.ss:143    1.             ?                                           -                  false          true                 {1, 3, }

Sankoff
    
    Total 0



Molecular
    
    Total 6
                                                                                                                                      
 Name         Initial Assignment         Transformation Cost Matrix         Gap Opening         Weight         class          
1.fas:0      Direct Optimization        tcm:(1,1)                          0                   1.             sequence      {}
1.fas:1      Direct Optimization        tcm:(1,1)                          0                   1.             sequence      {}
1.fas:2      Direct Optimization        tcm:(1,1)                          0                   1.             sequence      {}
1.fas:3      Direct Optimization        tcm:(1,1)                          0                   1.             sequence      {}
1.fas:4      Direct Optimization        tcm:(1,1)                          0                   1.             sequence      {}
40.aa:0      Direct Optimization        tcm:(1,1)                          0                   1.             sequence      {}

Likelihood
    
    Total 0




Ignored characters
    
Files
    


File
    

File Name : 40.aa

File Contents 


Characters



File
    

File Name : 30.ss

File Contents 


Characters

File Contents 


Trees



File
    

File Name : 1.fas

File Contents 


Characters







